assignment.js

```diff
@@ -112,8 +123,41 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 3:')
 
 // Add your code below this line
-
+function rockPaperScissors(playerA, playerB) {
```

Comment:
@SOyeniyi801 nice work covering all possible outcomes and using logical And (`&&`) in your conditions. Nice job maintaining good indentation throughout.